src/misc/DeviceConfig.ts

```diff
@@ -23,7 +23,10 @@ interface ConfigObject {
 	_themeId: ThemeId
 	_language: LanguageCode | null
 	_defaultCalendarView: Record<Id, CalendarViewType | null>
+	/** map from user id to a list of calendar grouproots*/
 	_hiddenCalendars: Record<Id, Id[]>
+	/** map from user id to a list of expanded folders (elementId)*/
+	expandedMailFolders: Record<Id, Id[]>
```

Comment:
should this be `Record<Id, Set<Id>>`?

Reply:
no, this is a plain JSON object, it can't have sets